src/main/java/com/sfuentes/pizzaShop/PizzaShop.java

```diff
@@ -0,0 +1,58 @@
+package com.sfuentes.pizzaShop;
+
+import lombok.Getter;
+
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+@Getter
+public class PizzaShop {
+
+  private final String pizzaSize;
+  private List<String> extraIngredients;
+  double totalPrice;
+  private final Map<String, Integer> small = Map.of("pizzaPreparationCost", 50, "baseCost", 10);
+  private final Map<String, Integer> medium = Map.of("pizzaPreparationCost", 60, "baseCost", 12);
+  private final Map<String, Integer> big = Map.of("pizzaPreparationCost", 80, "baseCost", 16);
```

Comment:
All of these could be a Pizza class with 2 properties, but you can leave it as it is.

Reply:
Good idea